docs/operations/hotplug_interfaces.md

```diff
@@ -0,0 +1,136 @@
+# Hotplug Network Interfaces
+KubeVirt now supports hotplugging network interfaces into a running Virtual
+Machine Instance (VMI). Hotplug is only supported for interfaces using the
+`virtio` model connected through
+[bridge binding](http://kubevirt.io/api-reference/main/definitions.html#_v1_interfacebridge).
+
+When a VM having hotplugged interfaces is rebooted, the hotplugged interfaces
```

Comment:
Could you drop this? I understand that this is significant part of the design, but it is not something I need to know as a user of this feature. For me it's obvious that interface I added will be still there after reboot.

It is also in conflict with "Persisting Hotplugged Interfaces" where you describe that an extra step is needed to persist the interface between reboots.

Reply:
Done.